.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
```

Comment:
Add jest as part of env.

Reply:
working on it

Reply:
Push changes after it is done and let at least one person review the PR.